packages/components/src/Hidden/Hidden.tsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
-import { Box, system } from '@marigold/system';
+import { createStyles, system } from '@marigold/system';
 
 type HiddenProps = {
+  variant?: string;
   show?: boolean;
 };
 
+const useStyles = createStyles('layout');
+
 export const Hidden = system<HiddenProps, 'span'>(
-  ({ variant = 'hidden', show = false, children, ...props }) => {
+  ({ as = 'span', variant = 'hidden', show = false, children, ...props }) => {
+    const classNames = useStyles({
+      variant,
+      display: show ? 'display' : 'none',
+    });
+    const Cmp = as;
```

Comment:
das brauchst du hier nicht, kannst in Z. 20 direkt `return <span ... >` schreiben